src/djikstra.cpp

```diff
@@ -1,6 +1,169 @@
 #include "djikstra.h"
 
+using namespace std;
 
-std::vector<Edge> Djikstra::findPath(int v1, int v2) {
-  return std::vector<Edge>();
+Djikstra::Djikstra(Graph *graph) {
+  if (!graph->isEdgeAggregated()) throw invalid_argument("Cannot give non edge aggregated graph to Djikstra's algorithm.");
```

Comment:
Why can't we? It doesn't affect the algorithm at all how the data was parsed/put into the graph